src/ui/settings/opencast.js

```diff
@@ -113,6 +114,9 @@ function OpencastSettings({ settingsManager }) {
               sx={{ ml: '10px', fontSize: '30px', verticalAlign: 'middle' }}
               spin={status === 'testing'}
             /> }
+            { hasRecording && status === 'saved' && (
+              <Link to="/" sx={{ ml: 3 }}>{t('settings-back-to-recording')}</Link>
```

Comment:
You should add "variant: 'styles.a'" to the sx attribute to style the link as defined in the theme.